Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/test/verification.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
It is more of a unitary test for us to assess the validity of the package. Mostly useful now as a regression test to make sure the behaviour works in the future. This could end up in py-qua-tools